pythoncz/models/github.py

```diff
@@ -63,27 +68,31 @@ def _get_issues_for_org(session, org_name):
             break
 
 
-def _enhance_issue(session, issue):
+def _issue_repo_slug(issue):
     repo_url_segments = issue['repository_url'].split('/')
-    repo_full_name = '{}/{}'.format(
-        repo_url_segments[-2],
-        repo_url_segments[-1]
-    )
+    org, repo = repo_url_segments[-2], repo_url_segments[-1]
+    return org, repo, '{}/{}'.format(org, repo)
+
 
-    res = session.get('https://api.github.com/repos/{}/issues/{}'.format(
-        repo_full_name,
+def _issue_url(issue):
+    org, repo, slug = _issue_repo_slug(issue)
+    return 'https://api.github.com/repos/{}/issues/{}'.format(
+        slug,
         issue['number']
-    ))
+    )
+
+
+def _enhance_issue(res):
     res.raise_for_status()
```

Comment:
previously, when a request could ahve not been finished, the whole process eneded up with exception in the middle. This stays the same. I can add a `RuntimeError` if res is `None` for better underestanding about what happened, but that's it. Or we can silently skip Nones to avoid 50x errors and just display less issues once in a while instead of Internal sever error.